src/track.jsx

```diff
@@ -68,7 +67,7 @@ var renderSlides = (spec) => {
     }
 
     var childStyle = getSlideStyle(assign({}, spec, {index: index}));
-    slides.push(cloneWithProps(child, {
```

Comment:
Care, I'm not sure this is wanted behaviour. Each slide should have his own props, see: https://facebook.github.io/react/docs/top-level-api.html#react.cloneelement